### PR TITLE
The position value was wrong

### PR DESCRIPTION
**Pull request by cisiqo**

### CODE REVIEW

**Comment by michallepicki** [comment]
Maybe there could be a way to create ex_unit doctests from `.md` files 🤔

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by axelson** [comment]
@michallepicki as of Elixir 1.15 there is! See `doctest_file/2`: https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#doctest_file/2

Unfortunately it isn't straightforward to apply in this case because many of `iex>` blocks in `lists-and-tuples.md` are not valid doc tests.

I'm very curious about the limitations of doctests so I've gone through and catalogued them.

`hd([])` results in a multi-line exception with blank lines which can't be doctested:
```elixir
iex> hd([])
** (ArgumentError) argument error
```

`IEx.Helpers.i` is not imported in doctests, `IEx.Helpers.i` actually returns `:"do not show this result in output"` and `...` can't be used to elide matching text:
```elixir
iex> i ~c"hello"
Term
  i ~c"hello"
Data type
  List
Description
  ...
Raw representation
  [104, 101, 108, 108, 111]
Reference modules
  List
Implemented protocols
  ...
```

`path/to/existing/file` doesn't point to a path within the elixir repository:
```elixir
iex> File.read("path/to/existing/file")
{:ok, "... contents ..."}
iex> File.read("path/to/unknown/file")
{:error, :enoent}
```

The blank lines mean these are separate doctests and you can't use `list` across them:
```elixir
iex> list = [1, 2, 3]
[1, 2, 3]

# This is fast as we only need to traverse `[0]` to prepend to `list`
iex> [0] ++ list
[0, 1, 2, 3]

# This is slow as we need to traverse `list` to append 4
iex> list ++ [4]
[1, 2, 3, 4]
```

So in short making these valid doctests would make it more difficult to convey the meaning that they're intended to.
